### PR TITLE
[MediaContent] Add face Insert API

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###
Add face insert method

### API Changes ###

 - TCSACR-249

### CODE REVIEW

**Comment by WonyoungChoi** [comment]
TCSACR-249 is in progress yet. Please merge this PR after the ACR has been confirmed.

**Comment by hsgwon** [comment]
> TCSACR-249 is in progress yet. Please merge this PR after the ACR has been confirmed.

The review process of TCSACR-249 is not started yet.
I'll merge this PR, when ACR is accepted.

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs [hunk 1]
```diff
@@ -140,5 +140,80 @@ public bool UpdateTag(string faceInfoId, string tag)
...
             }
         }
+
+        /// <summary>
+        /// Inserts a new face information to the database with the specified media ID, area, orientation.
+        /// </summary>
+        /// <privilege>http://tizen.org/privilege/content.write</privilege>
+        /// <param name="mediaId">The media ID to be associated with.</param>
+        /// <param name="area">The region of face in the media.</param>
+        /// <param name="orientation">The orientation of specified media.</param>
```
Comment (journey2w) [inline src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs]: /// <param name="orientation">The orientation of the specified media.</param>

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs [hunk 2]
```diff
@@ -140,5 +140,80 @@ public bool UpdateTag(string faceInfoId, string tag)
...
+        /// <returns>The <see cref="FaceInfo"/> containing the results.</returns>
+        /// <exception cref="InvalidOperationException">The <see cref="MediaDatabase"/> is disconnected.</exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="MediaDatabase"/> has already been disposed of.</exception>
```
Comment (journey2w) [inline src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs]: The <see cref="MediaDatabase"/> has already been disposed of what?

Suggestion:
  /// <exception cref="ObjectDisposedException">The <see cref="MediaDatabase"/> has already been disposed. </exception>

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs [hunk 3]
```diff
@@ -140,5 +140,80 @@ public bool UpdateTag(string faceInfoId, string tag)
...
+        /// </exception>
+        /// <exception cref="UnauthorizedAccessException">The caller has no required privilege.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public FaceInfo Insert(string mediaId, Rectangle area, Orientation orientation)
+        {
+            return Insert(mediaId, area, orientation, null);
+        }
+
+        /// <summary>
+        /// Inserts a new face information to the database with the specified media ID, area, orientation, and tag.
```
Comment (journey2w) [inline src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs]: Suggestion:
/// Inserts new face information to the database with the specified media ID, area, orientation, and tag.


Other rephrases are same as above.

File: src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs [hunk 4]
```diff
@@ -140,5 +140,80 @@ public bool UpdateTag(string faceInfoId, string tag)
                 Interop.Face.Destroy(handle);
             }
         }
+
+        /// <summary>
+        /// Inserts a new face information to the database with the specified media ID, area, orientation.
+        /// </summary>
+        /// <privilege>http://tizen.org/privilege/content.write</privilege>
+        /// <param name="mediaId">The media ID to be associated with.</param>
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs]: Dear @journey2w , I can't see the difference with original sentence in your suggestion.

How about the following sentence?
```
The UUID, which identify each media.
```

[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs]: If you intend to remove `<param name="mediaId">`, <param> should be described for all parameter as I know. If we don't, we will get warning message.

[the remark this answers is not in the document]
Comment (journey2w) [reply src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs]: @hsgwon,

I'm sorry my suggestion is not a completed sentence.
I mean `/// <param name="mediaId">The media ID to be associated with the face.</param>`

[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs]: OK. I'll update description.

**Comment by hsgwon** [comment]
@journey2w ,
I've uploaded new patch for your comment. Please review the PR.
https://github.com/Samsung/TizenFX/pull/943

**Pull request merged**